services/core-api/app/api/projects/project_summary/resources/project_summary_list.py

```diff
@@ -125,10 +131,14 @@ def post(self):
         if mine is None:
             raise NotFound('Mine not found')
 
+        current_app.logger.debug(f'mine: {mine_guid}')
         new_project = Project.create(mine, data.get('project_summary_title'),
                                      data.get('proponent_project_id'),
                                      data.get('mrc_review_required', False),
-                                     data.get('contacts', []))
+                                     data.get('contacts', []),
+                                     data.get('project_summary_lead_party_guid'))
```

Comment:
Not a big issue, but do you think we should change the name of that to "project_lead_party_guid", I realize the frontend would need a small change as well. I wonder if leaving it like this could be confusing to future devs without the context. Dealer's choice.